packages/cli-repl/package.json

```diff
@@ -78,13 +78,13 @@
     "@mongosh/types": "0.0.0-dev.0",
     "@segment/analytics-node": "^1.3.0",
     "ansi-escape-sequences": "^5.1.2",
-    "askcharacter": "^1.0.0",
-    "askpassword": "^1.2.4",
+    "askcharacter": "^2.0.4",
+    "askpassword": "^2.0.2",
     "escape-string-regexp": "^4.0.0",
     "is-recoverable-error": "^1.0.3",
     "js-yaml": "^4.1.0",
-    "mongodb-connection-string-url": "^3.0.0",
-    "mongodb-log-writer": "^1.4.0",
+    "mongodb-connection-string-url": "^3.0.1",
+    "mongodb-log-writer": "^1.4.2",
```

Comment:
Some of these versions got bumped twice because it turns out that CodeQL jobs with cron workflow triggers get disabled after 60 days of no activity in a repo and I didn't know that when doing the first set of releases 😕

Reply:
activity in the repo means commits to the main branch?

Reply:
I don't know, this is basically all the information available to me: 

<img width="1785" alt="Screenshot 2024-05-10 at 21 12 42" src="https://github.com/mongodb-js/mongosh/assets/899444/29b957ad-7ef1-470d-915c-d85261b4b8ff">